src/store/reducer/board.ts

```diff
@@ -137,6 +135,9 @@ export const boardReducer = (state: BoardState = {status: "unknown"}, action: Re
         status: "rejected",
       };
     }
+    case ActionType.CancelVoting: {
+      return state;
+    }
```

Comment:
You could use the default case for that. Both return the state without modifying

Reply:
Right :)